controllers/topics.controller.js

```diff
@@ -0,0 +1,7 @@
+const { fetchTopics } = require("../models/topics.model");
+
+exports.getTopics = (req, res, next) => {
+ fetchTopics().then((response) => {
+  res.status(200).send({ topics: response });
```

Comment:
You could clean this slightly by naming the response parameter `topics` and then short handing the object to `{ topics }`